src/tests/functional/shared_test_classes/src/subgraph/matmul_split_decompose.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/subgraph/matmul_split_decompose.hpp"
+
+#include "common_test_utils/data_utils.hpp"
+#include "common_test_utils/graph_comparator.hpp"
+#include "common_test_utils/node_builders/constant.hpp"
+#include "functional_test_utils/skip_tests_config.hpp"
+#include "openvino/pass/manager.hpp"
+#include "openvino/runtime/exec_model_info.hpp"
+#include "transformations/common_optimizations/matmul_split_decomposition.hpp"
```

Comment:
Could you please check if some headers are not needed? At least, we don't need `graph_comparator`, `skip_tests_config`, `manager` and `exec_model_info`

Reply:
Yes, `matmul_split_decomposition.hpp` and `data_utils.hpp` are also removed.